CopilotKit/packages/runtime/src/lib/runtime/remote-lg-action.ts

```
@@ -888,3 +791,170 @@ function getSchemaKeys(graphSchema: GraphSchema): SchemaKeys {
 function filterObjectBySchemaKeys(obj: Record<string, any>, schemaKeys: string[]) {
   return Object.fromEntries(Object.entries(obj).filter(([key]) => schemaKeys.includes(key)));
 }
+
+function getThreadIdFromArgs(args: ExecutionArgs) {
+  let threadId = args.threadId ?? randomUUID();
+  if (args.threadId && args.threadId.startsWith("ck-")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)